scripts/export.py

```diff
@@ -17,29 +17,22 @@
 PIN_FIELD = "PARID"
 SALE_KEY_FIELD = "SALEKEY"
 IS_OUTLIER_FIELD = "USER26"
-OUTLIER_TYPE_FIELD = "USER27"
+OUTLIER_REASON1_FIELD = "USER27"
+OUTLIER_REASON2_FIELD = "TODO1"
+OUTLIER_REASON3_FIELD = "TODO2"
 RUN_ID_FIELD = "USER28"
 
 OUTLIER_TYPE_CODES = {
-    "Not outlier": "0",
-    "Anomaly (high)": "1",
-    "Anomaly (low)": "2",
-    "Family sale (high)": "3",
-    "Family sale (low)": "4",
-    "High price (raw & sqft)": "5",
-    "High price (raw)": "6",
-    "High price (sqft)": "7",
-    "High price swing": "8",
-    "Home flip sale (high)": "9",
-    "Home flip sale (low)": "10",
-    "Low price (raw & sqft)": "11",
-    "Low price (raw)": "12",
-    "Low price (sqft)": "13",
-    "Low price swing": "14",
-    "Non-person sale (high)": "15",
-    "Non-person sale (low)": "16",
-    "PTAX-203 flag (High)": "17",
-    "PTAX-203 flag (Low)": "18",
+    None: "0",
```

Comment:
In the new spec we switched `"Not outlier"` to an NA value.

Reply:
**[Suggestion, non-blocking]** I think in this case we will most likely actually want to leave nulls as nulls, since iasworld will also expect nulls in this case (and not a code like `0`). I'll make a note to confirm this with Mirella, but in the meantime let's switch to leaving nulls as-is and we can adjust later if need be.